### PR TITLE
Autofocus on username field

### PR DESCRIPTION
**Pull request by maximal**
Useful UX improvement.

### CODE REVIEW

File: views/site/login.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
         ],
     ]); ?>
 
-        <?= $form->field($model, 'username') ?>
+        <?= $form->field($model, 'username')->textInput(['autofocus' => 'autofocus']) ?>
```
Comment (samdark) [inline views/site/login.php]: Shoundn't it be `'autofocus' => true`?

Comment (maximal) [inline views/site/login.php]: I usually prefer XML form for attributes (`autofocus="autofocus"`). But if you wish, we could use `autofocus`.

**Comment by maximal** [comment]
Should I change that to `'autofocus' => true`?
Which conventions on HTML5 do you use?

**Comment by maximal** [comment]
I can change it in advanced app also.

**Comment by samdark** [comment]
We're using just attribute w/o value for booleans. Would be great to get pull request for advanced template as well. Thanks!

**Comment by SilverFire** [comment]
Merged! Thank you!

**Comment by maximal** [comment]
You’re welcome.

**Pull request merged**